### PR TITLE
[CI:DOCS] updating build command documentation

### PR DESCRIPTION
**Pull request by wLMEB**
Added a few lines to fix the filenname ambiguity regarding `podmand build .` mentioned in #23039

Signed-off-by: Wanqi Li wanli@umass.edu

#### Does this PR introduce a user-facing change?

```release-note
None
```

### CODE REVIEW

**Comment by wLMEB** [comment]
I'm a college student majoring in computer science, and this is my first-ever contribution to an open-source project. I would really appreciate any feedback you could provide on my work. Please let me know if there are any areas where I can improve or any specific guidelines I should follow.

File: docs/source/markdown/podman-build.1.md.in [hunk 1]
```diff
@@ -13,7 +13,9 @@ podman\-build - Build a container image using a Containerfile
 Containerfiles or Dockerfiles and a specified build context directory. A
 Containerfile uses the same syntax as a Dockerfile internally. For this
 document, a file referred to as a Containerfile can be a file named
-either 'Containerfile' or 'Dockerfile'.
+either 'Containerfile' or 'Dockerfile' exclusively. Any file that has additional
+ extension attached will not be recognized by `podman build .` unless a `-f` flag is
+ used to specify the file.
```
Comment (Luap99) [inline docs/source/markdown/podman-build.1.md.in:18]: The extra space at the start of the line is not needed

**Review by Luap99 — COMMENTED** [review]
Please remove the extra `Signed-off-by: wLMEB <liwanqi0248@gmail.com>` line from the commit. The sign off with your real name is enough.

In general #23039 doesn't have a concrete resolution if podman should check this by default but as of now this matches what the code does today so this looks good to me.

**Comment by Luap99** [comment]
Please rebase from main (not merge) into your PR as this creates all the extra commits here as you can see that are not related to this PR at all. Also squash both of your commits into one, thanks.

**Comment by wLMEB** [comment]
Closing this PR in favor of a new one I opened: https://github.com/containers/podman/pull/23238

**Pull request closed without merging**